presentation/chapters/de-DE/iterators.chapter

```diff
@@ -0,0 +1,154 @@
+# Iteratoren
+
+---
+
+Rusts Iteratoren sind:
+
+* Faul
+* Pervasive
+* Potentiell unendlich
+
+---
+
+## Woher kommen sie?
+
+* Sammlungen wie `Vec<T>` haben eine `iter()`-Funktion, die einen Iterator erzeugt.
+* Dinge wie `std::net::TcpListener` die einen Iterator von `TcpStream`s bietet.
+* Iteratoren können auch auf andere Strukturen implementiert werden.
+
+---
+
+## Allgemeine Verwendung
+
+---
+
+## `next()`
+
+Iteratoren kann manuell fortgeschritten:
+
+```rust
+fn main() {
+    let items = vec![0, 1, 2];
+    let iterator = items.iter();
+    assert_eq!(iterator.next(), Some(0));
+    assert_eq!(iterator.next(), Some(1));
+    assert_eq!(iterator.next(), Some(1));
+    assert_eq!(iterator.next(), None);
+}
+```
+
+---
+
+## `map()`
+
+Transformieren Sie Elemente, wie sie ausgewertet werden:
```

Comment:
"Elemente können während der Auswertung transformiert werden."

Direkte Ansprache in Folien ("Sie", "Du") wirkt sehr odd.